mapreader/classify/classifier.py

```diff
@@ -1507,13 +1431,11 @@ def initialize_model(
             input_size = 299
 
         else:
-            raise ValueError("Invalid model name, exiting...")
+            raise NotImplementedError("[ERROR] Invalid model name.")  # CHECK THIS
```

Comment:
Suggesting a TODO here to make sure we catch it in the future

Reply:
I actually think I could remove this comment - wanted to check whether passing "resnet18" vs "resnet50" would give you the same model as output but am now confident it doesnt